pkg/util/util.go

```diff
@@ -229,3 +233,128 @@ func SplitQualifiedName(str string) (string, string) {
 func JoinQualifiedName(namespace, name string) string {
 	return path.Join(namespace, name)
 }
+
+type Route struct {
+	Interface   string
+	Destination net.IP
+	Gateway     net.IP
+	// TODO: add more fields here if needed
+}
+
+func getRoutes(file io.Reader) ([]Route, error) {
+	routes := []Route{}
+	scanner := bufio.NewReader(file)
+	for {
+		line, err := scanner.ReadString('\n')
+		if strings.HasPrefix(line, "Destination") || strings.HasPrefix(line, "Kernel") {
+			continue
+		}
+		if err == io.EOF {
```

Comment:
Prefer to check the error before looking at the result.  I.e. flip these 2 blocks.